src/common/libraries/MessageLib.sol

```diff
@@ -11,12 +11,12 @@ import {AssetId} from "src/common/types/AssetId.sol";
 enum MessageType {
     /// @dev Placeholder for null message type
     _Invalid,
-    // -- Root messages
+    // -- Pool independent messages
     ScheduleUpgrade,
     CancelUpgrade,
     RecoverTokens,
-    // -- Pool manager messages
     RegisterAsset,
+    // -- Pool dependent messages
```

Comment:
I think we can reserve some values here if in the future we need more pool independent messages. WDTY?

We now anyways need to block of MESSAGE_LENGTH, so I think it's feasible

Reply:
Yeah that seems like a good idea.

Reply:
[b7f3577](https://github.com/centrifuge/protocol-v3/pull/407/commits/b7f35770f0a79253abb5ff349f2faa1e9861b562)